util/log/context.go

```diff
@@ -0,0 +1,94 @@
+// Copyright 2015 The Cockroach Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+// implied. See the License for the specific language governing
+// permissions and limitations under the License. See the AUTHORS file
+// for names of contributors.
+//
+// Author: Tobias Schottdorf
+
+package log
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/cockroachdb/cockroach/util/log/logfield"
+	"golang.org/x/net/context"
+)
+
+// RoachContext wraps a context.Context, providing additional sugar.
+type RoachContext interface {
```

Comment:
How about just calling this `Context`? The `Roach` part feels redundant given this is part of the `cockroach/util/log` package.